master/buildbot/steps/trigger.py

```diff
@@ -198,13 +210,21 @@ def addBuildUrls(self, rclist):
     @defer.inlineCallbacks
     def run(self):
         schedulers_and_props = yield self.getSchedulersAndProperties()
+        
+        # To be back compatible we need to differ between old and new style
+        # schedulers_and_props can either consist of 2 elements or 3 elements tuples
+        
+        # If the user getSchedulersAndProperties funcion does not specify the importance (red len(tup)==2), we make all schedulers important
+        schedulers_and_props = [tup if len(tup)==3 else (tup[0], tup[1], False) for tup in schedulers_and_props]
```

Comment:
so you are suggesting something like:

if tuple:
  old_style
else:
 #dict
?